apps/ejabberd/src/mod_mam_odbc_async_pool_writer.erl

```diff
@@ -158,9 +170,9 @@ stop_muc(Host) ->
 %% API
 %%====================================================================
 
-start_workers(Host) ->
-    [start_worker(WriterProc, N, Host)
-     || {N, WriterProc} <- worker_names(Host)].
+start_workers(Host, PoolSize) ->
+  [start_worker(WriterProc, N, Host)
```

Comment:
Please maintain the indentation style used throughout the project. 4 spaces per indentation level, no tabs, only spaces.